pages/manual/myplanet.md

```diff
@@ -33,7 +33,7 @@ At the login screen click the Gear icon top right corner
 
 ![Server Setup](images/myplanet/login_setup.png)
 
-On the resulting screen, enter the local server IP Address in the form of a URL. URL will be what you open on browser to use planet. Unless you received special instructions, enter: http://192.168.2.1. Also, enter the Pin from Step 1 above
+On the resulting screen, enter the local server IP Address in the form of a URL. URL will be what you open on browser to use planet. For example, you should use your laptop ip address in the form of a URL. Also, enter the Pin from Step 1 above
```

Comment:
You need to also put IP address there as example so its easier to understand

Reply:
Thanks! that makes sense. I added my ip address in the sentence.

Reply:
Wait! I will edit the sentence a little more. I will push this new commit now.